pkg/features/kube_features.go

```diff
@@ -631,6 +631,7 @@ const (
 	// owner: @ehashman
 	// alpha: v1.21
 	// beta: v1.22
+	// ga: v1.27
```

Comment:
is there any e2e tests that needs to be promoted to Conformance?

Reply:
Just work on that https://github.com/kubernetes/kubernetes/pull/116091 and find this PR.